redis/db/redisbp/hooks.go

```diff
@@ -79,20 +82,54 @@ func (h SpanHook) startChildSpan(ctx context.Context, cmdName string) context.Co
 		name,
 		tracing.SpanTypeOption{Type: tracing.SpanTypeClient},
 	)
+	redisprom.ActiveRequests.With(prometheus.Labels{
+		redisprom.ClientNameLabel: h.ClientName,
+		redisprom.TypeLabel:       h.Type,
+		redisprom.CommandLabel:    cmdName,
+		redisprom.DeploymentLabel: h.Deployment,
+		redisprom.DatabaseLabel:   "", // We don't have that info
```

Comment:
we can just add this field to `SpanHook` struct and get it, right?

Reply:
passed through when I have it